src/witan/models/dem/ccm/mort/mortality_mvp.clj

```diff
@@ -8,6 +8,10 @@
             [incanter.core :as i]))
 
 (defn create-popn-at-risk
+  "Takes datasets with population, deaths, and births. Returns a new
+  dataset with population at risk in the :popn column. The population
+  at risk is the births data joined with a subset of the population
+  dataset that has been aged on."
```

Comment:
The population at risk is the births data (age 0) joined with the aged on population
dataset (age 1 to 90). Subset data for years less than or equal to the latest year of deaths data"